src/bundle/Resources/translations/search.en.xliff

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
-  <file date="2017-11-07T21:29:28Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
+  <file date="2017-11-09T13:48:46Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
```

Comment:
Can we clean up this PR from unrelated translation files? I think during extraction you can limit domains.

Reply:
Yes, I will try